mindsdb/integrations/handlers/symbl_handler/README.md

```diff
@@ -1,90 +1,84 @@
-# Symbl Handler
-
-Symbl handler for MindsDB provides interfaces to connect to Symbl via APIs and pull conversation data into MindsDB.
-
 ---
-
-## Table of Contents
-
-- [Symbl Handler](#symbl-handler)
-  - [Table of Contents](#table-of-contents)
-  - [About Symbl](#about-symbl)
-  - [Symbl Handler Implementation](#symbl-handler-implementation)
-  - [Symbl Handler Initialization](#symbl-handler-initialization)
-  - [Implemented Features](#implemented-features)
-  - [Example Usage](#example-usage)
-
+title: Symbl
+sidebarTitle: Symbl
 ---
 
-## About Symbl
-
-Transform unstructured audio, video and text data into structured insights, events and knowledge.
-
-
-## Symbl Handler Implementation
+This documentation describes the integration of MindsDB with [Symbl](https://symbl.ai/), a platform with with state-of-the-art and task specific LLMs, enables businesses to analyze multi-party conversations at scale.
+This integration allows MindsDB to process conversation data and extract insights from it.
 
-This handler was implemented using the `symbl` library that makes http calls to https://docs.symbl.ai/docs/welcome.
+## Prerequisites
 
-## Symbl Handler Initialization
+Before proceeding, ensure the following prerequisites are met:
 
-The Symbl handler is initialized with the following parameters:
+1. Install MindsDB locally via [Docker](/setup/self-hosted/docker) or [Docker Desktop](/setup/self-hosted/docker-desktop).
```

Comment:
Please replicate the updates from the doc page into this readme file.

Also, please note that in READMEs, we need to specify the full link to docs - for example, instead of `/setup/self-hosted/docker`, we need to use `https://docs.mindsdb.com/setup/self-hosted/docker`.